readme.md

```diff
@@ -21,12 +21,12 @@ Follow
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://Victor-Mirandas.github.io/layout_stars/)
+- [TEST REPORT LINK](https://Victor-Mirandas.github.io/layout_stars/report/html_report/)
```

Comment:
check your deployment (or ask for a help at fe_chat) since DEMO link leads to readme.md file (not working page), TEST link leads to 404 page